src/expressions.md

```diff
@@ -600,6 +600,12 @@ within this loop may exit out of this loop or return control to its head.
 See [break expressions](#break-expressions) and [continue
 expressions](#continue-expressions).
 
+The return value of a `loop` expression is `!`, which can be coerced to any
+type. However, if there is a `break` expression in the loop, the return value
```

Comment:
Does the `break` expression have to also break for the loop it's in? For instance, I would expect `'inner` to still be considered diverging here? I'm not actually sure though.

```rust
'outer loop {
  'inner loop {
    break 'outer;
  }
}
```